c-deps/libroach/fmt.h

```diff
@@ -0,0 +1,19 @@
+// Copyright 2017 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied.  See the License for the specific language governing
+// permissions and limitations under the License.
+
+#pragma once
+
+#include <google/protobuf/stubs/stringprintf.h>
+
+namespace fmt = google::protobuf;
```

Comment:
@bdarnell: I ended up switching to this, resulting in invocations of `fmt::StringPrintf(...)`. I know this means that you could technically be using `fmt::MessageLite` but I don't think that's much of an issue. I've left the non-formatting uses of `google::protobuf` intact.